src/radical/entk/task.py

```diff
@@ -586,6 +610,50 @@ def luid(self):
 
         return '%s.%s.%s' % (p_elem, s_elem, t_elem)
 
+    # --------------------------------------------------------------------------
+    #
+    def annotate(self, inputs: Union[Dict, List, str, None] = None,
+                       outputs: Union[List, str, None] = None) -> None:
+        """
+        Adds dataflow annotations with provided input and output files,
+        and defines dependencies between tasks.
+
+        Attributes:
+            inputs (list, optional): List of input files. If file is produced
+                by previously executed task, then the corresponding input
+                element is provided as a dictionary with task instance as a key.
```

Comment:
"If file is produced by previously executed task, then the corresponding input element is provided as a dictionary with task instance as a key" -> If a file is produced by the previously executed task, then the corresponding input element is provided as a dictionary with the task instance as a key?

Reply:
done